Makefile

```diff
@@ -68,56 +91,79 @@ vet:
 
 # Generate code
 generate: controller-gen
-	${CONTROLLER_GEN} object:headerFile="./hack/boilerplate.go.txt" paths="./pkg/apis/..."
+	$(CONTROLLER_GEN) object:headerFile="./hack/boilerplate.go.txt" paths="./pkg/apis/..."
 
 build-controller-image:
 	bazel run cmd/forklift-controller:forklift-controller-image
 
 push-controller-image: build-controller-image
-	$(CONTAINER_CMD) tag cmd/forklift-controller:forklift-controller-image ${CONTROLLER_IMAGE}
-	$(CONTAINER_CMD) push ${CONTROLLER_IMAGE}
+	$(CONTAINER_CMD) tag bazel/cmd/forklift-controller:forklift-controller-image $(CONTROLLER_IMAGE)
```

Comment:
If we want to be precise it should be `localhost/bazel/...`. Or at least that's how it looks on my env. Or is that perhaps different when Docker is used instead of Podman?

This also applies to all the other occurrences in Makefile.

Reply:
> If we want to be precise it should be `localhost/bazel/...`. Or at least that's how it looks on my env. Or is that perhaps different when Docker is used instead of Podman?

yeah, Miguel and I also suspected it depends on Docker vs. Podman - since I got the same image tag as you with Podman

Reply:
Prepending `localhost` to the image name is something specific to podman, docker does not do that at all. On the other hand, podman is smart enough when a "hostless" image is specified and it prepends `localhost` to it.

Reply:
I also realized that the `CONTAINER_CMD` variable is not necessary, we should use `docker` command always as  it's the only runtime supported by bazel's container rules. To use `podman` under the hood `podman-docker` package is necessary so the `docker` command exists to be used by bazel.

Reply:
> On the other hand, podman is smart enough when a "hostless" image is specified and it prepends `localhost` to it.

right, I just verified that these changes work for me with podman